aiohttp/payload.py

```diff
@@ -278,6 +296,30 @@ def __init__(self, value,
             content_type=content_type, encoding=encoding, *args, **kwargs)
 
 
+class AsyncIterablePayload(Payload):
+
+    def __init__(self, value, *args, **kwargs):
+        assert isinstance(value, AsyncIterable), \
```

Comment:
Shouldn't here be a regular `if not isinstace: raise TypeError` check?

Reply:
It should probably but other classes in the module use `assert` statements.

Reply:
Ok.